app/scripts/modules/core/src/pipeline/config/stages/travis/modal/addParameter.html

```diff
@@ -0,0 +1,45 @@
+<div modal-page>
+  <modal-close dismiss="$dismiss()"></modal-close>
+  <div class="modal-header">
+    <h3 data-purpose="modal-header">Add Parameter</h3>
+  </div>
+
+  <form role="form" class="container-fluid" novalidate name="addParameterFrom">
+    <div class="modal-body">
+      <div class="form-group row">
+        <div class="col-sm-3 sm-label-right">Key</div>
+        <div class="col-sm-9">
+          <input type="text"
+                 class="form-control input-sm "
+                 ng-model="parameter.key"
+                 placeholder="enter a parameter key"
+                 required>
+          </input>
+        </div>
+      </div>
+      <div class="form-group row">
+        <div class="col-sm-3 sm-label-right">Value</div>
+        <div class="col-sm-9">
+          <input type="text"
+                 class="form-control input-sm "
+                 ng-model="parameter.value"
+                 placeholder="enter a parameter value"
+                 required>
+          </input>
+        </div>
+      </div>
+    </div>
+
+    <div class="modal-footer">
+      <a href class="btn btn-default" ng-click="$dismiss()">Cancel</a>
+      <button type="submit"
+              class="btn btn-primary"
+              data-purpose="submit"
+              ng-click="addParameter.submit()"
+              ng-disabled="addParameter.$invalid"
+        >
```

Comment:
Move this to the end of the previous line to make it equal to the style used in the rest of the file